docs/application/dotnet/guides/app-management/component-port.md

```diff
@@ -0,0 +1,469 @@
+# Component Port
+
+Tizen components of component-based application can communicate with each other using component ports. Components can send and receive serializable objects through component port communication.
+
+The main feature of the `Tizen.Applications.ComponentBased.ComponentPort` class include:
+- Managing a Component Port
+  You can set up Component ports to [send and receive requests](#port) between components of component-based application with the `Tizen.Applications.ComponentBased.ComponentPort` class.
+
+- Using a Component Task
+  You can set up Component tasks to [run tasks](#task) to wait for events from other components with the `Tizen.Applications.ComponentBased.ComponentTask` class.
+
+## Prerequisites
+
+To enable your component to use the component port functionality:
+
+1.  You need 2 components to communicate with each other through the component port.
+2.  To use the methods and properties of the [Tizen.Applications.ComponentBased.ComponentPort](https://samsung.github.io/TizenFX/latest/api/Tizen.Applications.ComponentBased.ComponentPort.html) and [Tizen.Applications.ComponentBased.ComponentTask](https://samsung.github.io/TizenFX/latest/api/Tizen.Applications.ComponentBased.ComponentPort.html) classes, include the [Tizen.Applications.ComponentBased](https://samsung.github.io/TizenFX/latest/api/Tizen.Applications.ComponentBased.html) namespace in your component:
```

Comment:
Link is breaking, please verify

Reply:
The reference(doxygen) is not updated not yet. 
This feature will be added after Tizen 6.5 M1 release.

Reply:
I've updated it.
https://samsung.github.io/TizenFX/API9/api/Tizen.Applications.ComponentBased.ComponentPort.html

The link should be "https://samsung.github.io/TizenFx/API9/api/...".